### PR TITLE
Validation: alt-text-missing fixes.

### PR DESCRIPTION
**Pull request by brittmsantos**
Various alt-text-missing fixes.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=642) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=642) before we can accept your contribution.<br/><br/>:x: brittmsantos [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=642)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetDocs?pullRequest=642) it.</sub>

File: aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-5.md [hunk 1]
```diff
@@ -25,15 +25,15 @@ In the past chapter, we were loading data from our database and displaying it. I
...
+![Screenshot of the Add Controller dialog with the Controller box and the Template, Model class, and Data context class dropdowns highlighted.](mvc-music-store-part-5/_static/image1.png)
 
 When you click the Add button, you'll see that the ASP.NET MVC 3 scaffolding mechanism does a good amount of work for you:
 
 - It creates the new StoreManagerController with a local Entity Framework variable
 - It adds a StoreManager folder to the project's Views folder
 - It adds Create.cshtml, Delete.cshtml, Details.cshtml, Edit.cshtml, and Index.cshtml view, strongly typed to the Album class
 
-![](mvc-music-store-part-5/_static/image2.png)
+![Screenshot of the StoreManagerController window opened in Microsoft Visual Web Developer 2010 Express after creation.](mvc-music-store-part-5/_static/image2.png)
```
Comment (dereklh7) [inline aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-5.md:36]: Write as "Store Manager Controller".

File: aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-5.md [hunk 2]
```diff
@@ -53,19 +53,19 @@ The modified view code will appear as follows:
 
 Now run the application and browse to /StoreManager/. This displays the Store Manager Index we just modified, showing a list of the albums in the store with links to Edit, Details, and Delete.
 
-![](mvc-music-store-part-5/_static/image3.png)
+![Screenshot of the Store Manager index showing a list of albums in the store with links to Edit, Details, and Delete.](mvc-music-store-part-5/_static/image3.png)
 
 Clicking the Edit link displays an edit form with fields for the Album, including dropdowns for Genre and Artist.
 
-![](mvc-music-store-part-5/_static/image4.png)
+![Screenshot of the Edit dialog showing drop-down fields for Genre and Artist and text boxes for Artist, Title, Price, and AlbumArtUrl.](mvc-music-store-part-5/_static/image4.png)
```
Comment (dereklh7) [inline aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-5.md:60]: Write as "Album Art U R L".

File: aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-5.md [hunk 3]
```diff
@@ -53,19 +53,19 @@ The modified view code will appear as follows:
...
 
 Click the "Back to List" link at the bottom, then click on the Details link for an Album. This displays the detail information for an individual Album.
 
-![](mvc-music-store-part-5/_static/image5.png)
+![Screenshot of the Details dialog showing the set values for Genre, Artist, Title, Price, and AlbumArtUrl.](mvc-music-store-part-5/_static/image5.png)
```
Comment (dereklh7) [inline aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-5.md:64]: Write as "Album Art U R L".

File: aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-5.md [hunk 4]
```diff
@@ -164,9 +164,9 @@ To test this out, run the application and browse to /StoreManager/Create/ - this
 
 Fill in some values and click the Create button to submit the form.
 
-![](mvc-music-store-part-5/_static/image7.png)
+![Screenshot of the Create Form showing the Genre and Artist dropdown fields and the Title, Price, and AlbumArtUrl text boxes.](mvc-music-store-part-5/_static/image7.png)
```
Comment (dereklh7) [inline aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-5.md:167]: Write as "Album Art U R L".

File: aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-7.md [hunk 5]
```diff
@@ -37,31 +37,31 @@ Change the namespace for the Controller and Model classes so they begin with Mvc
 
 The updated solution should look like the following:
 
-![](mvc-music-store-part-7/_static/image1.png)
+![Screenshot of the Solution Explorer pane with AccountController.cs, AccountModels.cs, and the files in the /Views/Account/ folder highlighted in red.](mvc-music-store-part-7/_static/image1.png)
```
Comment (dereklh7) [inline aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-7.md:40]: Write as "Account Controller dot C S, Account Models dot CS, and the files in the / Views / Account / folder..."

**Comment by brittmsantos** [comment]
@Rick-Anderson 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**